presto-docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -13,14 +13,36 @@ Synopsis
 Description
 -----------
 
-Set a session property value.
+Set a session property value or a catalog session property.
+
+Catalog session properties use the name of a catalog as a prefix in front of a
+session property supported by a connector.
 
 Examples
 --------
 
+The following example enables optimized hash generation.
+
 .. code-block:: sql
 
     SET SESSION optimize_hash_generation = true;
+
+An example of the usage of catalog session properties is the :doc:`Accumulo
```

Comment:
We normally link with the original title. Let's rephrase this a bit.
```
An example of the usage of catalog session properties is the :doc:`/connector/accumulo`,
which supports a property named ``optimize_locality_enabled``. If your Accumulo catalog
is named ``data``, you would use the following to set the catalog session property::

    SET SESSION data.optimize_locality_enabled = false;

Reply:
if you link with the original doc link you end up getting title case usage in the middle of the sentence. I think this is wrong and should be avoided where possible. I updated the rest.